### DISCUSSION TITLE
SMILES without property info

### DISCUSSION THREAD

**Discussion by rajathsalegame** [rdkit/rdkit#3837] [Q&A] [answered]
Is there an easy way to convert a mol that has stored properties to SMILES without the properties getting encoded into the SMILES? I need the mol to retain properties, because for some strange reason, adding atom map numbers as a property to the mol results in reversed chirality when I write the SMILES. That bug is referenced here: https://github.com/rdkit/rdkit/issues/3835

**Comment by greglandrum** [comment, marked as answer]
I've commented on #3835 itself, but that's not a bug.

If you want to output a SMILES that doesn't include atom map numbers, you can just remove the atom map numbers from the molecule by setting them to zero:
```
>>> m = Chem.MolFromSmiles('[CH3:1][OH:2]')
>>> Chem.MolToSmiles(m)
'[CH3:1][OH:2]'
>>> for atom in m.GetAtoms():
...     atom.SetAtomMapNum(0)
... 
>>> Chem.MolToSmiles(m)
'CO'
```

**Reply by rajathsalegame** [reply]
Thanks!
